guvnor-bom/pom.xml

```diff
@@ -164,6 +164,12 @@
       </dependency>
       <dependency>
         <groupId>org.guvnor</groupId>
+        <artifactId>guvnor-ala-build-maven</artifactId>
+        <version>${version.org.kie}</version>
+        <classifier>tests</classifier>
```

Comment:
@psiroky I see, thanks. in this case the <classifier>tests</classifier> is being used.